src/site/content/en/blog/appcache-removal/index.md

```diff
@@ -0,0 +1,222 @@
+---
+title: Preparing for AppCache removal
+subhead: Chrome 85 removes support for AppCache by default. Most developers should migrate off AppCache now, and not wait any longer.
+authors:
+  - jeffposnick
+description: Details of Chrome's and other browsers' plans to remove AppCache.
+date: 2020-05-18
+updated: 2020-05-18
+scheduled: true
+tags:
+  - appcache
+  - blog
+  - chrome84
+  - deprecation
+  - origin-trial
+  - removal
+  - service-worker
+hero: hero.jpg
+alt: An old-fashioned storage container.
+origin_trial:
+  url: https://developers.chrome.com/origintrials/#/view_trial/1776670052997660673
+---
+
+Following up on [previous announcements](https://blog.chromium.org/2020/01/appcache-scope-restricted.html), support for [AppCache](https://developer.mozilla.org/en-US/docs/Web/HTML/Using_the_application_cache) will be removed from Chrome and other Chromium-based browsers. We encourage developers to migrate off of AppCache now, rather than waiting any longer.
```

Comment:
```suggestion
Following up on [previous announcements](https://blog.chromium.org/2020/01/appcache-scope-restricted.html), support for [AppCache](https://developer.mozilla.org/docs/Web/HTML/Using_the_application_cache) will be removed from Chrome and other Chromium-based browsers. We encourage developers to migrate off of AppCache now, rather than waiting any longer.
```
Removes hardcoding to English version of page